stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/querychannels/DefaultChatEventHandlerTest.kt

```diff
@@ -0,0 +1,165 @@
+package io.getstream.chat.android.offline.querychannels
+
+import io.getstream.chat.android.client.events.NotificationAddedToChannelEvent
+import io.getstream.chat.android.client.events.NotificationMessageNewEvent
+import io.getstream.chat.android.client.events.NotificationRemovedFromChannelEvent
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.Filters
+import io.getstream.chat.android.offline.randomChannel
+import io.getstream.chat.android.offline.randomMember
+import io.getstream.chat.android.offline.randomMessage
+import io.getstream.chat.android.offline.randomUser
+import io.getstream.chat.android.test.randomInt
+import io.getstream.chat.android.test.randomString
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.amshove.kluent.`should be equal to`
+import org.junit.jupiter.api.Test
+import java.util.Date
+
+internal class DefaultChatEventHandlerTest {
+
+    @Test
+    fun `When received NotificationAddedToChannelEvent and the channel is not present, it should be added`() {
```

Comment:
Nice tests! 👍 
One thing. We have [naming convention](https://github.com/GetStream/InternalAndroidGuidelines#testing) for tests. According to it this test should have the following name:
`Given the channel is not present When handle NotificationAddedToChannelEvent Should add this channel`